stratosphere/dmnt/source/cheat/impl/dmnt_cheat_vm.cpp

```diff
@@ -46,8 +46,13 @@ namespace ams::dmnt::cheat::impl {
     void CheatVirtualMachine::OpenDebugLogFile() {
         #ifdef DMNT_CHEAT_VM_DEBUG_LOG
         CloseDebugLogFile();
-        R_ABORT_UNLESS(fs::OpenFile(std::addressof(m_debug_log_file), "sdmc:/atmosphere/cheat_vm_logs/debug_log.txt"));
+        fs::EnsureDirectory("sdmc:/atmosphere/cheat_vm_logs");
+        char log_path[fs::EntryNameLengthMax + 1] = "sdmc:/atmosphere/cheat_vm_logs/debug_log.txt";
```

Comment:
why are you putting this path on the stack? This code doesn't really look like idiomatic ams code.

Also, this should abort on failure.